inst/shiny/DiagnosticsExplorer/R/Results.R

```diff
@@ -1,88 +1,57 @@
-createDatabaseDataSource <- function(connection, 
-                                     resultsDatabaseSchema, 
-                                     vocabularyDatabaseSchema = resultsDatabaseSchema) {
-  return(list(connection = connectionPool,
-              resultsDatabaseSchema = resultsDatabaseSchema,
-              vocabularyDatabaseSchema = vocabularyDatabaseSchema))
-}
-
-
-createFileDataSource <- function(premergedDataFile, envir = new.env()) {
-  load(premergedDataFile, envir = envir)
-  return(envir)
-}
+# createDatabaseDataSource <- function(connection, 
```

Comment:
its always best to remove code rather than commenting it. (makes the reviewers job easier ;))